pkg/controller/jobs/jobset/jobset_webhook.go

```diff
@@ -57,9 +67,41 @@ func (w *JobSetWebhook) Default(ctx context.Context, obj runtime.Object) error {
 	log.V(5).Info("Applying defaults", "jobset", klog.KObj(jobSet))
 
 	jobframework.ApplyDefaultForSuspend(jobSet, w.manageJobsWithoutQueueName)
+
+	if canOverrideManagedBy(jobSet.Spec.ManagedBy) {
+		localQueueName, found := jobSet.Labels[constants.QueueLabel]
+		if !found {
+			return nil
+		}
+		clusterQueueName, err := w.queues.ClusterQueueFromLocalQueue(queue.QueueKey(jobSet.ObjectMeta.Namespace, localQueueName))
+		if err != nil {
+			return err
+		}
+		clusterQueue, found := w.cache.GetClusterQueue(clusterQueueName)
+		if !found {
+			return nil
+		}
+		for admissionCheckName := range clusterQueue.AdmissionChecks {
+			admissionCheck, ok := w.cache.GetAdmissionCheck(admissionCheckName)
+			if !ok {
+				continue
+			}
+			if admissionCheck.Controller == multikueue.ControllerName {
+				log.V(5).Info("Overriding ManagedBy", "jobset", klog.KObj(jobSet), "managedBy", multikueue.ControllerName)
```

Comment:
```suggestion
				log.V(5).Info("Defaulting managedBy", "jobset", klog.KObj(jobSet), "managedBy", multikueue.ControllerName)
```
nit: since from 0.5.1 JobSet does not set the managedBy (so it will be nil).